### ISSUE TITLE
Amazing content. Could you please let us know the effective batch size used during the training ?

### ISSUE THREAD

**Issue by shamanez** [HLTCHKUST/VG-GPLMs#1]

**Comment by shamanez** [comment]
In the paper, it says, 120 batch size with 4 GPUs.  So can each GPU load up to 30 examples or is it with gradient accumulation steps ?

**Comment by TysonYu** [comment]
Hi, The 120 is computed by 4GPUs * 5 gradient accumulation * 6 batch size each GPU
